### PR TITLE
Under the hood!

### LINKED ISSUE #3: Add A New Slide
## We Need More Slides
Grab another octocat and create a new slide from scratch.
If you need help, check out the Appendix in the training manual.

### PR DESCRIPTION
**Pull request by toddwalstad-eaton**
This resolves #3

### CODE REVIEW

File: _slides/14-toddwalstad-eaton.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Most people use and IDE that hides all this, this is good to learn what is happening under the hood!
```
Comment (ViktorStamenov-eaton) [inline _slides/14-toddwalstad-eaton.md:6]: That's good

**Review by ViktorStamenov-eaton — APPROVED** [review]
Good to go

File: _slides/14-toddwalstad-eaton.md (hunk 1 above)
Comment (toddwalstad-eaton) [reply _slides/14-toddwalstad-eaton.md:6]: @ViktorStamenov-eaton thanks, can you please approve?

**Comment by toddwalstad-eaton** [comment]
> Good to go

You beat me to it!

**Pull request merged**